### DISCUSSION TITLE
Moving to blazorbootstrap from Blazorise

### DISCUSSION THREAD

**Discussion by cisberner** [vikramlearning/blazorbootstrap#422] [Q&A]
Hi all, In my Blazor project I used Blazorise, but I was wondering if someone has moved from Blazorise to blazorbootstrap, and if that would need a lot of work?

**Comment by gvreddy04** [comment]
@cisberner I'm not sure, but before migrating, please check the BlazorBootstrap demos. We developed all the components with real-world application scenarios in mind. We add more enhancements in every release. Feel free to ask for support if you are migrating from Blazorise to BlazorBootstrap.

Demos link: https://demos.blazorbootstrap.com/
